.shipyard.e2e.yml

```diff
@@ -5,3 +5,4 @@ nodes: control-plane worker
 clusters:
   cluster1:
   cluster2:
+    nodes: control-plane worker worker
```

Comment:
This should go in the project that needs it though, shipyard itself has no real need to test with that many modes

Reply:
ack. Will remove.

Reply:
Done